### ISSUE TITLE
Regex failing silently

### ISSUE THREAD

**Issue by karthikb351** [karthikb351/OpenCAD#1]
Pulling the `content_category` from [Paragraph 94 from Tuesday, the 29th April, 1947](https://github.com/karthikb351/OpenCAD/blob/d019c66b4beac6fbce6b75fc4b28689b05752ded/src/sample.txt#L109) fails silently.

``` json
{
  "content_category": [
    "education",
    "school## In these clauses dealing with the fundamental rights",
    "we find the word 'school' and also the words 'educational institutions' being used at different places",
    "leading one to think that some distinction is intended. I would like it to be clearly stated that by school we mean any educational institution. I am referring to clause 18 sub-clause (2) where it is stated-- \"No minority whether based on religion",
    "community or language shall be discriminated against in regard to the admission into State educational institutions",
    "nor shall any religious instruction be compulsorily imposed on them.\"    ##minorities",
    "education",
    "school",
    "discrimination",
    "freedom of religion## ||29|| Here the words used are \"State educational institutions\". In sub-clause (3) (a) it is laid down-- \"All minorities whether based on religion",
    "community or language shall be free in any Unit to establish and administer educational institutions of their choice.\"  ##minorities",
    "education",
    "school",
    "discrimination",
    "freedom of religion## ||30|| Here we have the words \"educational institutions\". And in sub-clause (3) (b) the word 'schools' is used-- \"The State shall not",
    "while providing State aid to schools",
    "discriminate against schools under the management of minorities whether based on religion",
    "community or language.\"        ##minorities",
    "education",
    "school",
    "discrimination",
    "freedom of religion"
  ]
}
```

**Comment by kishore-narendran** [comment]
On it.

**Comment by kishore-narendran** [comment]
Fixed it in e6eedcdf1f26cd9ac1bd71b7219aec2db8219149
The regex parser needs to take care of duplicates in content category and duplicates in the mentioned names. And duplicates of name and name's showing up.
I am on that.

**Comment by kishore-narendran** [comment]
And this bug arose, because there is supposed to be one content category at the end of every paragraph, according to the formatting they mentioned in the doc.
